src/js/status.js

```diff
@@ -0,0 +1,6 @@
+export const updateCompletionStatus = (taskId, status, tasks) => {
+  tasks[taskId].completed = status;
+  return tasks;
+};
+
+export const clearAllCompleted = (tasks) => tasks.filter(task => !task.completed);
```

Comment:
- [ ] Currently after deleting all completed tasks the index of the remaining tasks left doesn't get reordered I kindly suggest you fix this.